definition_validator/validator.cc

```diff
@@ -535,8 +535,12 @@ void validateSuperClass(core::Context ctx, const core::ClassOrModuleRef sym, con
 
     if (auto e = ctx.state.beginError(core::Loc(sym.data(ctx)->loc().file(), classDef.declLoc),
                                       core::errors::Resolver::NonClassSuperclass)) {
-        e.setHeader("The super class `{}` of `{}` does not derive from `{}`", superClass.show(ctx), sym.show(ctx),
+        auto superClassFqn = superClass.show(ctx);
+        e.setHeader("The super class `{}` of `{}` does not derive from `{}`", superClassFqn, sym.show(ctx),
                     core::Symbols::Class().show(ctx));
+        for (const auto loc : superClass.data(ctx)->locs()) {
+            e.addErrorLine(loc, "{} defined here", superClassFqn);
+        }
```

Comment:
This is technically unrelated to the change. It just helped debugging some stuff, and I figured I'd keep it.